src/test/java/io/stargate/sgv2/jsonapi/testresource/StargateTestResource.java

```diff
@@ -250,6 +258,14 @@ private String getAuthToken(String host, int authPort) {
     }
   }
 
+  public int getMaxCollectionsPerDBOverride() {
```

Comment:
Defined as overridable methods for sub-classes (per-test custom test resources) to override.